docs/adr/tilekit.md

```
@@ -0,0 +1,70 @@
+- GET /agent/stop - Stops and kills the Pi agent
+- GET /agent/state - Returns the current state of Agent
+- GET /agent/end-session - Stops the current session gracefully
+- POST /agent/prompt - Sends the user prompt
+  - ```json
+    {
+      "message": "create a todo for today"
+    }
+    ```
+  - The response of this API will SSE events as the clients will be streaming the response and the type of event & data will be following Pi's types.
+
+  - The events will follow Pi events as [here](https://github.com/earendil-works/pi/blob/main/packages/coding-agent/docs/rpc.md#events).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)